src/test/java/org/sonar/generic/metrics/TestGenericMetricsSensor.java

```diff
@@ -188,6 +193,6 @@ public void whenExecutingAndCanReadFileAndFileInJsonNotFoundShouldNotPublishMetr
     readMetrics();
 
     sensor.execute(this.sensorContext);
-    verify(sensorContext, times(0)).newMeasure();
+    verify(sensorContext, times(1)).newMeasure();

```

Comment:
Even though the file cannot be found, a project level metric is supplied, hence 1 not 0